internal/controller/controllers/crd_utils.go

```diff
@@ -63,6 +66,13 @@ func (u *CRDUtils) CreateAgentCR(ctx context.Context, log logrus.FieldLogger, ho
 				Namespace: infraEnv.Namespace,
 			},
 		}
+
+		// Update 'kube_key_namespace' in Host
+		if err1 := db.Model(&common.Host{}).Where("id = ?", hostId).Update(
```

Comment:
to say the truth i would prefer if we can use some existent function for it. Or lets add new one to hosts api. I am not sure it is the best idea to have db object inside controllers utils

Reply:
sure, I'll add it to hostAPI then.